src/main/kotlin/no/nav/familie/ef/sak/behandling/BehandlingRepository.kt

```diff
@@ -75,4 +77,18 @@ interface BehandlingRepository : RepositoryInterface<Behandling, UUID>, InsertUp
     @Query("""SELECT id FROM gjeldende_iverksatte_behandlinger WHERE stonadstype=:stønadstype""")
     fun finnSisteIverksatteBehandlinger(stønadstype: Stønadstype): Set<UUID>
 
+    // language=PostgreSQL
```

Comment:
Burde vi ha denne i en egen respository-klasse (og flytte testen for denne og)?
`GjeldendeBarn` som den returnerer ligger under iverksett.oppgaveforbarn